cpp/CMakeLists.txt

```diff
@@ -181,7 +181,9 @@ endif()
 
 set(CUGRAPH_SOURCES
     src/detail/shuffle_vertices.cu
-    src/detail/shuffle_vertex_pairs.cu
+    src/detail/shuffle_vertex_pairs_int32_int32.cu
+    src/detail/shuffle_vertex_pairs_int32_int64.cu
+    src/detail/shuffle_vertex_pairs_int64_int64.cu
```

Comment:
What's the rationale behind this? Is this to cut compile time?

Reply:
We should be more consistent on this to keep our codebase maintainable.

Reply:
Yes, this was to cut compile time.  It was the one file that was being constantly recompiled while I was testing this that took a long time.

I have been picking these up when I see them to split them.  I think ultimately we need to re-evaluate how we are doing type dispatching so that we can generally improve compile time.

Reply:
Yeah... we should re-evaluate this, but AFAIK, this isn't the file with the longest compile time, and just splitting this file looks a bit odd. No problem in doing this for local development, but I guess pushing this change to the main repo (without really agreeing on how to tackle compile time issue) sounds a bit premature.